custom-metrics-stackdriver-adapter/README.md

```diff
@@ -129,18 +151,29 @@ by a simple name, as defined in [custom metric naming rules].
        podID := *podIdFlag
        ```
 
-     - `container_name = ""`
-     - `project_id`, `zone`, `cluster_name` - can be obtained by your application from
-        [metadata server]. You can use Google Cloud compute metadata client to get
-        these values, example in Go:
+     - For monitored resource `gke_container`, `container_name` should be set to
+       `""` to indicate that the metric is associated with a pod, not a
+       particular container.
+     - `project_id`, `zone`, `location`, `cluster_name` - can be obtained by your
+       application from [metadata server]. You can use Google Cloud compute
+       metadata client to get these values, example in Go:
 
        ```go
        import gce "cloud.google.com/go/compute/metadata"
 
        project_id, err := gce.ProjectID()
+       // the zone where your application runs
+       // used in legacy resource model
        zone, err := gce.Zone()
+       // cluster location can be different than your application zone in
+       // clusters spanning across multiple zones
+       // used in new resource model
+       location, err := gce.InstanceAttributeValue("cluster-location")
        cluster_name, err := gce.InstanceAttributeValue("cluster-name")
        ```
+     - `namespace_id` and `instance_id` (for **legacy resource model**) are not
+       used by Custom Metrics - Stackdriver Adapter, but still it's recommended
```

Comment:
Why is it recommended?

Reply:
Added a short rationale. All resource labels are required, so it should be quite obvious that it's better to set them to correct values than to some random strings.